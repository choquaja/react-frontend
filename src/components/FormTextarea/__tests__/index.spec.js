import React from 'react';
import { shallow } from 'enzyme';
import FormTextarea from '../';

describe('FormTextarea', () => {
  it('renders without crashing', () => {
    shallow(<FormTextarea />);
  });

  it('renders with block set true', () => {
    const component = shallow(<FormTextarea block />);
    expect(component).toMatchSnapshot();
  });

  it('renders with full set true', () => {
    const component = shallow(<FormTextarea full />);
    expect(component).toMatchSnapshot();
  });
  it('renders with large set true', () => {
    const component = shallow(<FormTextarea large />);
    expect(component).toMatchSnapshot();
  });
  it('renders with small set true', () => {
    const component = shallow(<FormTextarea small />);
    expect(component).toMatchSnapshot();
  });
  it('renders with grouped set true', () => {
    const component = shallow(<FormTextarea grouped />);
    expect(component).toMatchSnapshot();
  });
});
