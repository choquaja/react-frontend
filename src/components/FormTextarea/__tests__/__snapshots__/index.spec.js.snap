// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormTextarea renders with block set true 1`] = `
<textarea
  className="FormTextarea__FormTextarea-hoGdLG eWXqSE"
/>
`;

exports[`FormTextarea renders with full set true 1`] = `
<textarea
  className="FormTextarea__FormTextarea-hoGdLG giOorR"
/>
`;

exports[`FormTextarea renders with grouped set true 1`] = `
<textarea
  className="FormTextarea__FormTextarea-hoGdLG iQpwBC"
/>
`;

exports[`FormTextarea renders with large set true 1`] = `
<textarea
  className="FormTextarea__FormTextarea-hoGdLG lfwnu"
/>
`;

exports[`FormTextarea renders with small set true 1`] = `
<textarea
  className="FormTextarea__FormTextarea-hoGdLG kEDprJ"
/>
`;
